webapp/public/index.html

```
@@ -0,0 +1,357 @@
+                  aria-selected="false" aria-controls="view-pending" data-view="pending">
+            Pending URLs
+          </button>
+        </li>
+      </ul>
+
+      <!-- Actions -->
+      <div class="nav-actions">
+        <span class="demo-badge" id="demo-badge" style="display:none">
+          Demo Mode
+        </span>
+        <button class="theme-toggle" id="theme-toggle"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)